L1Trigger/L1THGCal/src/HGCalTriggerTools.cc

```diff
@@ -170,7 +170,7 @@ int HGCalTriggerTools::zside(const DetId& id) const {
 
 int HGCalTriggerTools::thicknessIndex(const DetId& id, bool tc) const {
   if (isScintillator(id)) {
-    throw cms::Exception("IncompatibleDetId") << "Trying to get the sensor thickness of a scintillator detid";
```

Comment:
This function is used elsewhere and `kScintillatorPseudoThicknessIndex_` has no meaning in other parts of the code. The exception should be kept, calling `thicknessIndex` for a scintillator detid is not valid.